lib/core-integration/src/Test/Integration/Scenario/API/Shelley/StakePools.hs

```diff
@@ -144,6 +148,22 @@ spec = describe "SHELLEY_STAKE_POOLS" $ do
     let listPools ctx stake = request @[ApiStakePool] ctx
                 (Link.listStakePools stake) Default Empty
 
+    it "STAKE_POOLS_MAINTENANCE_01 - \
+        \trigger GC action when metadata source = direct" $ \ctx -> runResourceT $ do
+        updateMetadataSource ctx "direct"
+        verifyMetadataSource ctx FetchDirect
+        triggerMaintenanceAction ctx "gc_stake_pools"
+        eventually "GC Status shows as NotApplicable" $ do
```

Comment:
We should have an `eventually` that isn't 90+ seconds. Triggering this should be in the realm of <10s usually.

Reply:
ok, added eventually with 10s timeout.